lib/src/functions/map.dart

```diff
@@ -21,8 +22,25 @@ final global = UnmodifiableListView([
 ]);
 
 /// The Sass map module.
-final module = BuiltInModule("map",
-    functions: [_get, _merge, _remove, _keys, _values, _hasKey]);
+final module = BuiltInModule("map", functions: [
+  _get, _merge, _remove, _keys, _values, _hasKey, //
+  BuiltInCallable.overloadedFunction("set", {
```

Comment:
Please follow the pattern used for other functions by declaring this as a top-level field and then including it by name in the `functions` list.